src/index.js

```diff
@@ -0,0 +1,4 @@
+import "./css/main.css";
```

Comment:
We should keep the "ES6 supported comment here".

Reply:
done